ProcessMaker/processmaker
# QA Observation Bugfix/FOUR-13236: Error in console when clicking on a process box

## Solution
- Reported console error was fixed

## How to Test
- Go to Process Catalogue
- Click on some Available Process Category
- Click on Any Process
- Check browser console

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-13236

## Code Review Checklist
ci:deploy
ci:next
